src/coreclr/src/jit/simdashwintrinsiclistxarch.h

```diff
@@ -0,0 +1,102 @@
+//                  ISA          Function name               NumArg                                                                                                                                                                       Instructions                                                                                                                                                                                              Flags
+//                                                                      {TYP_BYTE,                           TYP_UBYTE,                          TYP_SHORT,                          TYP_USHORT,                         TYP_INT,                            TYP_UINT,                           TYP_LONG,                           TYP_ULONG,                          TYP_FLOAT,                          TYP_DOUBLE}
+// *****************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************************
+//  Vector<T> Intrinsics
+SIMD_AS_HWINTRINSIC(VectorT128,  AndNot,                     2,         {NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE2_AndNot,                     NI_SSE_AndNot,                      NI_SSE2_AndNot},                       SimdAsHWIntrinsicFlag::NeedsOperandsSwapped)
+SIMD_AS_HWINTRINSIC(VectorT128,  Equals,                     2,         {NI_SSE2_CompareEqual,               NI_SSE2_CompareEqual,               NI_SSE2_CompareEqual,               NI_SSE2_CompareEqual,               NI_SSE2_CompareEqual,               NI_SSE2_CompareEqual,               NI_SSE41_CompareEqual,              NI_SSE41_CompareEqual,              NI_SSE_CompareEqual,                NI_SSE2_CompareEqual},                 SimdAsHWIntrinsicFlag::None)
+SIMD_AS_HWINTRINSIC(VectorT128,  GreaterThan,                2,         {NI_SSE2_CompareGreaterThan,         NI_AVX2_CompareGreaterThan,         NI_SSE2_CompareGreaterThan,         NI_AVX2_CompareGreaterThan,         NI_SSE2_CompareGreaterThan,         NI_AVX2_CompareGreaterThan,         NI_SSE42_CompareGreaterThan,        NI_SSE42_CompareGreaterThan,        NI_SSE_CompareGreaterThan,          NI_SSE2_CompareGreaterThan},           SimdAsHWIntrinsicFlag::NotTableDriven)
```

Comment:
For `LONG` and `ULONG`, we can actually support it on even lower than SSE42 via emulation and do for `GT_SIMD`, I didn't port that logic yet, however, and it will fallback to the `GT_SIMD` path on the older hardware (or if the user disables `SSE4.2` path via `COMPlus`)

Reply:
Fixed